src/rabbit_msg_store_ets_index.erl

```diff
@@ -68,8 +68,8 @@ delete_object(Obj, State) ->
     true = ets:delete_object(State #state.table, Obj),
     ok.
 
-delete_by_file(File, State) ->
-    MatchHead = #msg_location { file = File, _ = '_' },
+cleanup_undefined_file(State) ->
```

Comment:
How about `clean_up_entries_with_undefined_file`, to clarify that the `file` field should be `undefined`?